assets/performanceprofile/tuned/openshift-node-performance-amd-x86

```diff
@@ -1,5 +1,7 @@
 [main]
-Summary=Platform specific tuning for AMD x86
+summary=Platform specific tuning for AMD x86
+
+include=openshift-node-performance
```

Comment:
Nope, will not work. The operator does not know the architecture of the node and so cannot select the proper active profile this way.

You need to use the conditional include in the same way we do for the RT kernel

Reply:
Though it is a bit messy I did see a way to use only f functions to handle the conditional import like the rt kernel profile uses, and they do seem to be evaluting correctly on my test server

```
2024-06-13 15:41:04,411 DEBUG    tuned.profiles.functions.functions: ${f:lscpu_check:Vendor ID:\s*GenuineIntel:intel:AuthenticAMD:amd:ARM:arm} expands to: 'intel'
2024-06-13 15:41:04,415 DEBUG    tuned.profiles.functions.functions: ${f:lscpu_check:Architecture:\s*x86_64:x86:aarch64:aarch64} expands to: 'x86'
```